camdaemon.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+import datetime
+import time
+
+import picamera2
+
+import config
+
+
+def picam2_setup(picam2: picamera2.Picamera2) -> None:
+    picam2.start_preview(picamera2.Preview.NULL)
```

Comment:
To fix #3 - we can add a `try/except`  (I have a prototype working on now out of date code).  Do you want to add it here at a low level or put it in `main()`?  

Here's a local solution (faux code, untested, might just work?)

```suggestion
    try:
        picam2.start_preview(picamera2.Preview.NULL)
    except RuntimeError as r_err:
        print("got RuntimeError when trying to call take_photo:", r_err)
        os._exit(1)
    except IndexError as e_err:
        print("Camera not found maybe? got IndexError when trying to call take_photo:", e_err)
        os._exit(1)
```

Reply:
Good thought! Adding to main to catch the with initialization. Also made a ticket to look at exception handling more.

Reply:
cools - resolving this comment thread and we can chase it down elsewhere